Sources/FrenchRepublicanCalendarCore/FrenchRepublicanDateOptions.swift

```diff
@@ -18,9 +18,17 @@ public struct FrenchRepublicanDateOptions {
     public var romanYear: Bool
     public var variant: Variant
     
+    /// If true, formatted dates in Sansculottides are things like "1 Sansculottides" instead of a holiday name.
+    public var treatSansculottidesAsAMonth: Bool
+    
     public init(romanYear: Bool, variant: Variant) {
+        self.init(romanYear: romanYear, variant: variant, treatSansculottidesAsAMonth: false)
+    }
+    
+    public init(romanYear: Bool, variant: Variant, treatSansculottidesAsAMonth: Bool) {
```

Comment:
Could use a default value of false for treatSansculottidesAsAMonth instead of duplicating here too